README.md

```diff
@@ -1,10 +1,13 @@
-# Kanban Board
+# TV Shows database
 
-> We are creating a kanban Board as part of Microverse project
+> This is the app where you could see information about shows as well as leave likes and comments on your favorite shows.
 
-![screenshot](./readme_archive.png)
+![screenshot](./screenshot-1.png)
+![screenshot](./screenshot-2.png)
 
-"To Do List" is a simple app that help you record and manage tasks. You can create a new task, update it, mark it as complete, order in terms of priority and delete them when completed.
+For this project we used TV Maze API for listing the shows and Involvement API to save the likes and comments.
+
```

Comment:
- [ ] Instead of just writing clone the project, please mention the command how to clone. Think about the recruiter who doesn't have a technical background and wants to test the project locally.

```
git clone https://github.com/patriciachrysy/kanban-board.git

cd kanban-board
```
and then all the steps which you already mentioned like `npm install` and `npm start`.

Reply:
Ok, done